### PR TITLE
Add missing example DAG in documentation

### PR DESCRIPTION
**Pull request by pankajastro**
closes: #490 

Here, I'm documenting an example DAG for recently merged PR going forward we will try to include example DAG documentation in the operator/sensor implementation PR itself

### CODE REVIEW

**Comment by rajaths010494** [comment]
LGTM

File: docs/providers/microsoft/azure/sensors/wasb.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+WASB Blob Sensor Async
+""""""""""""""""""""""
+
+
+To polls asynchronously for the existence of a blob in a WASB container
```
Comment (pankajkoti) [inline docs/providers/microsoft/azure/sensors/wasb.rst:5]:
```suggestion
To poll asynchronously for the existence of a blob in a WASB container.
```

File: docs/providers/microsoft/azure/sensors/wasb.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+:class:`~astronomer.providers.microsoft.azure.sensors.wasb.WasbBlobSensorAsync`.
+
+.. exampleinclude:: /../astronomer/providers/microsoft/azure/example_dags/example_wasb_sensors.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_sensor_wasb_blob_sensor_async]
+    :end-before: [END howto_sensor_wasb_blob_sensor_async]
+
+
+To polls asynchronously for the existence of a blob having the given prefix in a WASB container
```
Comment (pankajkoti) [inline docs/providers/microsoft/azure/sensors/wasb.rst:15]:
```suggestion
To poll asynchronously for the existence of a blob having the given prefix in a WASB container
```

**Review by pankajkoti — APPROVED** [review]
Thanks Pankaj for adding these

**Review by rajaths010494 — APPROVED** [review]

**Pull request merged**